uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/AbsenceType.php

```diff
@@ -212,6 +212,15 @@ public static function getValuesArray($id) {
     return $absenceType;
   }
 
+  /**
+   * Returns a list of all possible categories for absence type
+   *
+   * @return array|bool
```

Comment:
Does this return a boolean also, under what conditions?

Reply:
Return type of `buildOptions` is `array|bool` which explains the reason for this return type.